gitbot.js

```diff
@@ -51,14 +52,21 @@ function findGitRepos(repos, depth, callback) {
 function getCommitsFromRepos(repos, days, callback) {
   let cmts = [];
   async.each(repos, (repo, repoDone) => {
+    localGitUsername = ''
+    try {
+      gitUtilsRepo = git.open(repo);
+      localGitUsername = gitUtilsRepo.getConfigValue('user.name');
```

Comment:
What is `getConfigValue` returns en empty name? Maybe you should add an fallback!?
```js
localGitUsername = gitUtilsRepo.getConfigValue('user.name') || gitUsername;
```